python/README.md

```diff
@@ -0,0 +1,95 @@
+# Python
+
+Python hits a sweet spot between short shell scripts and large applications.
+We use Python 3.6+ for largish scripts and smallish libraries.
+
+## Links
+
+- [Python 3 documentation](https://docs.python.org/3/)
+    - `!py` with [DuckDuckGo](https://duckduckgo.com)
```

Comment:
It wasn't that hard for me to figure out what you meant here, but maybe it would be clearer if this read:

```
- Search for `!py` in [DuckDuckGo](https://duckduckgo.com)
```

Reply:
I made this change.